From: Andreas Cord-Landwehr <cola@uni-paderborn.de>
Subject: Do not compile incompatible licensed SVN plugin
Forwarded: no
Author: Andreas Cord-Landwehr <cola@uni-paderborn.de>
Last-Update: 2013-06-09

Description: SVN plugin uses GPL-3+ license while KDevPlatform
  contains files with incompatible GPL-2 license.
--- a/plugins/CMakeLists.txt
+++ b/plugins/CMakeLists.txt
@@ -35,12 +35,12 @@ if(Boost_FOUND)
     macro_optional_add_subdirectory(classbrowser)
 endif(Boost_FOUND)
 
-find_package(SubversionLibrary)
-macro_log_feature( SUBVERSION_FOUND "Subversion" "Support for Subversion integration" "http://subversion.tigris.org" FALSE "1.3.0" "The subversion libraries are needed for the Subversion support" )
+#find_package(SubversionLibrary)
+#macro_log_feature( SUBVERSION_FOUND "Subversion" "Support for Subversion integration" "http://subversion.tigris.org" FALSE "1.3.0" "The subversion libraries are needed for the Subversion support" )
 
-if(SUBVERSION_FOUND)
-    macro_optional_add_subdirectory(subversion)
-endif(SUBVERSION_FOUND)
+#if(SUBVERSION_FOUND)
+#    macro_optional_add_subdirectory(subversion)
+#endif(SUBVERSION_FOUND)
 
 if(QJSON_FOUND)
     macro_optional_add_subdirectory(reviewboard)
